frontend/src/metabase/visualizations/components/ChartTooltip/utils.ts

```diff
@@ -18,3 +19,34 @@ export const formatValueForTooltip = ({
     type: "tooltip",
     majorWidth: 0,
   });
+
+export const groupExcessiveTooltipRows = (
+  rows: TooltipRowModel[],
+  maxRows: number,
+) => {
+  if (rows.length <= maxRows) {
+    return rows;
+  }
+
+  const groupStartingFromIndex = maxRows - 1;
+  const result = rows.slice();
+  const rowsToGroup = result.splice(groupStartingFromIndex);
+
+  rowsToGroup.reduce(
+    (grouped, current) => {
+      if (
+        typeof current.value === "number" &&
+        typeof grouped.value === "number"
+      ) {
+        grouped.value += current.value;
+      }
+      return grouped;
+    },
+    {
+      color: color("text-light"),
+      name: `Other`,
+      value: 0,
+      formatter: rowsToGroup[0].formatter,
+    },
+  );
+};
```

Comment:
I don't think this function is used? accidental duplicate of the one inside `stackedDataTooltip/utils` ?